src/test/java/com/zufar/onlinestore/AddressDtoConverterTest.java

```diff
@@ -0,0 +1,75 @@
+package com.zufar.onlinestore;
+
+import com.zufar.onlinestore.user.converter.AddressDtoConverter;
+import com.zufar.onlinestore.user.dto.AddressDto;
+import com.zufar.onlinestore.user.entity.Address;
+import org.instancio.Instancio;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import static io.smallrye.common.constraint.Assert.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@SpringBootTest
+class AddressDtoConverterTest {
+
+    @Autowired
+    private AddressDtoConverter converter;
+    private Address address;
+    private AddressDto addressDto;
+
+    @BeforeEach
+    void setUp() {
+        address = Instancio.create(Address.class);
+        addressDto = Instancio.create(AddressDto.class);
+    }
+
+    @Test
+    void shouldCheckAddressDoesNotEqualToNull() {
+        assertNotNull(address);
```

Comment:
Do you check Instancio?

Reply:
yes, this case is checking that entity shouldn't equal to null